src/Controls/tests/Xaml.UnitTests/Issues/Gh2304.xaml.cs

```diff
@@ -0,0 +1,21 @@
+using System;
+using System.Collections.Generic;
+using Microsoft.Maui.Controls;
+using Microsoft.Maui.Controls.Core.UnitTests;
+using NUnit.Framework;
+
+namespace Microsoft.Maui.Controls.Xaml.UnitTests
+{
+	public partial class Gh2304
+	{
+		[TestFixture]
+		class Tests
+		{
+			[Test]
+			public void XamlCDoesntFail()
+			{
+				new Gh2304();
+			}
```

Comment:
we usually test for both XamlC and other parser (see other tests). also, the naming we use is no longer Ghxxx (ghxxxx was the name used to refer to GitHub issue in the old Xamarin.Forms repository. I tend to use Mauixxxx nowadays)

Reply:
This has been done, and the test is now named `Maui2304Closed` (to make room for `Maui2304Open`, if I could ever figure out how to test open generic type syntax, which I can't, so we might want to rename to just `Maui2304`?).